fern/pages/fern-docs/content/customize-api-ref.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: Customize API Reference layout
+description: Customize your API Reference's naming, ordering, and structure.  
+---
+
+When you include an API (either the Fern or an OpenAPI definition) in your `docs.yml` file, you can customize how the endpoints and sub-packages are displayed in the sidebar navigation. By default, the reference will generate a navigation hierarchy based on the structure of the API spec, but several customizations can be configured.
+
+## Ordering the API Reference
+
+### Flattening sections
+To remove the API Reference title and display the section contents, set `flattened` to `true`.
+
+```yaml title="docs.yml"
+navigation: 
+  - api: API Reference
+    flattened: true
+```
+
+<Frame>
+<img src="flattened.png" alt="Flattened API Reference" />
+</Frame>
+
+### Alphabetizing endpoints and sections
+To sort all sections and endpoints alphabetically, unless explicitly ordered in `layout`, set `alphabetized` to `true`.
+
+
+```yaml title="docs.yml"
+navigation: 
+  - api: API Reference
+    alphabetized: true
```

Comment:
Seeing an example screenshot + preview link would be preferable as a user.

Reply:
Here, my thought was to bias towards being brief whenever possible. I don't think "seeing" alphabetization is critical to understanding what the result is

Reply:
agree with you here! I get what it'll do